airflow/plugins/operators/amplitude_to_flattened_json.py

```diff
@@ -0,0 +1,51 @@
+"""Module for exporting data from Amplitude and adding to the data warehouse"""
+import os
+import gzip
+import zipfile
+from io import BytesIO, StringIO
+
+import requests
+import pandas as pd
+
+from airflow.models import BaseOperator
+
+DATE_FORMAT = "%Y%m%dT%H"
+
+
+def amplitude_to_df(
+    start: str,
+    end: str,
+    api_key: str = os.environ.get("CALITP_AMPLITUDE_BENEFITS_API_KEY"),
+    secret_key: str = os.environ.get("CALITP_AMPLITUDE_BENEFITS_SECRET_KEY"),
```

Comment:
It seems a little awkward / unnecessary to have the default values do the environment lookup?

I think it's more natural for the defaults be `None` and do the lookup inside the function body only if needed.